sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_greengrass_example.ipynb

```diff
@@ -0,0 +1,1202 @@
+{
```

Comment:
<p>Don't hardcode AMIs. What if a customer is running this example in a region besides these 6? Maybe <code>describe_images()</code>  can be used to retrieve them based on the boto3 session region? https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.describe_images</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Good call-out. Done</p>